js/statements/use.js

```diff
@@ -0,0 +1,22 @@
+function GenerateUse(options = {}) {
+    return Diagram([
+        AutomaticStack([
+            Keyword("USE"),
+            Expression("database-name"),
+            Optional(Expression(".schema-name"), "skip")
```

Comment:
The `.` should be a separate `Keyword`

Reply:
Fixed on https://github.com/duckdb/duckdb-web/pull/688/commits/355fc58fed2093e4fb6dfc57b2e3c80f16cc126b